lms/templates/notifications/digest_footer.html

```diff
@@ -0,0 +1,32 @@
+<table cellpadding="0" cellspacing="0" style="color:black; font-weight:400; font-size:12px;line-height:20px" width="100%">
+    <tbody>
+        <tr>
+            <td>
+                <table width="100%">
+                    <tbody>
+                        <tr>
+                            <td width="25%" align="left" style="padding: 0">
+                                <img src="{{ logo_url }}" style="width: auto;" height="24" alt="Logo"/></a>
+                            </td>
+                            <td width="75%" align="right" style="padding: 0">
+                                <table>
+                                    <tbody>
+                                        <tr>
+                                            {% for social_name, social_data in social_media.items %}
+                                                <td style="padding: 0">
+                                                    <a href="{{social_data.url}}">
+                                                        <img src="{{social_data.icon}}" style="max-height: 24px; max-width: 24px; margin-left: 0.6rem" alt="{{social_name}}" />
```

Comment:
We have set them to empty dict [here](https://github.com/mitodl/ol-infrastructure/blob/af8fe68c70eb079e6df8760ad38bf74161839dc1/src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl#L500-L501). If we set them, it will show the clickable icons.